default_robot_padding: 0.0
default_robot_scale: 1.0
default_object_padding: 0.0
default_attached_padding: 0.0
default_robot_link_padding: {    main_body    : 0.05,
                                 link0       : 0.05,
                                 bl          : 0.05,
                                 link1       : 0.05,
                                 link2       : 0.05,
                                 link3       : 0.05,
                                 link4       : 0.05,
                                 link5       : 0.05
 }
